pynetbox_data_uploader/tests/test_read_csv.py

```diff
@@ -0,0 +1,33 @@
+from pytest import fixture, raises
+from lib.user_methods.csv_to_netbox import CsvToNetbox
+
+
+@fixture(name="instance")
+def instance_fixture():
+    """
+    This method calls the class being tested.
+    :return: The class with mock arguments.
+    """
+    mock_url = "mock_url"
+    mock_token = "mock_token"
+    return CsvToNetbox(mock_url, mock_token)
+
+
+def test_check_file_path_valid(instance):
+    """
+    This test checks that the method doesn't raise any errors for valid filepaths.
+    """
+    mock_file_path = "."
+    instance.check_file_path(mock_file_path)
+
+    # with not raises(FileNotFoundError):
+    #     instance.read_csv(mock_file_path)
```

Comment:
Might want to remove these left over comments if not being implemented?

Reply:
👍